php-config/Slate/CBL/Tasks/Task.config.d/search-conditions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Slate\CBL\Tasks;
+
+use \Emergence\People\Person;
+
+$personTableAlias = Person::getTableAlias();
+
+$searchConditions = [
+    'Title' => [
+        'qualifiers' => ['any', 'title'],
+        'points' => 2,
+        'sql' => 'Title LIKE "%%%s%%"'
+    ],
+    'Created' => [
+        'qualifiers' => ['created'],
+        'points' => 1,
+        'sql' => 'CAST(Created AS Date) = "%s"'
+    ],
+    'ParentTaskTitle' => [
+        'qualifiers' => ['parenttasktitle', 'parenttask'],
+        'points' => 1,
+        'join' => [
+            'className' => Task::class,
+            'localField' => 'ParentTaskID',
+            'foreignField' => 'ID',
+            'aliasName' => 'ParentTask'
+        ],
+        'sql' => 'ParentTask.Title LIKE "%%%s%%"'
+    ],
+    'Creator' => [
+        'qualifiers' => ['creatorfullname', 'creator'],
+        'points' => 1,
+        'join' => [
+            'className' => Person::class,
+            'localField' => 'CreatorID',
+            'foreignField' => 'ID',
+            'aliasName' => $personTableAlias
+        ],
+        'sql' => 'CONCAT('.$personTableAlias.'.FirstName, " ", '.$personTableAlias.'.LastName) LIKE "%%%s%%"'
+    ]
+];
+
+Task::$searchConditions = array_merge(Task::$searchConditions, $searchConditions);
```

Comment:
@nbey php-config files shouldn't be used to provide part of the base implementation of a class. If you really need to set some config dynamically, you can do that by overriding the magic __classLoaded function emergence implements for classes (it gets run after config has been loaded but before the autoloader returns control to the first place using the class)

Also, I believe you can set the `'sql'` option for searchConditions to a callable like `[__CLASS__, 'staticMethodName']` and not have to set any config dynamically. Shouldn't the aliasName be getting calculated automatically the same way already?